src/Microsoft.SqlTools.ServiceLayer/FileBrowser/FileBrowserOperation.cs

```diff
@@ -248,6 +260,26 @@ internal bool FilterFile(string fileName, string[] masks)
             return false;
         }
 
+        /// <summary>
+        /// Creates file tree for a scoped path
+        /// Removes the top level file tree so only use this if you intend to do that
+        /// </summary>
+        /// <param name="expandPath">path to consider at top level node for tree</param>
+        private void ScopeFileTreeToPath(string expandPath)
+        {
+            this.fileTree = new FileTree();
+            FileTreeNode node = new FileTreeNode()
+            {
+                Name = expandPath,
+                FullPath = expandPath,
+                IsExpanded = true
+            };
+
+            this.fileTree.RootNode.AddChildNode(node);
```

Comment:
is this adding '/var/opt/mssql/data' as a child node of the root in addition to any other nodes that came back from enumerating the root?

Reply:
/var/opt/mssql/data is the this.fileTree.RootNode. This is adding the bak files and any child folders to it.